packages/core/stories/form-field/form-field.doc.stories.mdx

```diff
@@ -62,6 +68,18 @@ The Form Field Label can be positioned to the left of the form control via the p
 
 Place your label to the left of the control if your application is limited in vertical real estate or if you have a very data heavy, dense form, where it’s preferable to have as many fields visible as possible without scrolling.
 
+## Form Field Label Question
+
+The Form Field Label can be styled as a question by setting the `question` prop to `true`.
```

Comment:
what does it mean to be styled as a question? I feel this is confusing. Can we have a variant=bold or something?

Reply:
I also think this is easily achievable with additional styling applied by the users. 
Should it be just an example?

Reply:
it's going to be a common use case cc @bhoppers2008 - I would go against variant=bold as it goes against primary/secondary we have elsewhere

Reply:
the wording can definitely be improved though

Reply:
Of course, you're right about the variant=bold (wasn't being serious about it tbh), but I would personally avoid `question`.